plugins/services/src/js/components/ServiceStatusProgressBar.js

```diff
@@ -0,0 +1,80 @@
+import React, { PropTypes } from "react";
+import { Tooltip } from "reactjs-components";
+
+import StringUtil from "#SRC/js/utils/StringUtil";
+import ProgressBar from "#SRC/js/components/ProgressBar";
+import Pod from "../structs/Pod";
+import Service from "../structs/Service";
+import ServiceTree from "../structs/ServiceTree";
+import ServiceStatus from "../constants/ServiceStatus";
+
+class ServiceStatusProgressBar extends React.Component {
+  getTooltipContent() {
+    const { service } = this.props;
+    const { tasksRunning } = service.getTasksSummary();
+    const instancesTotal = service.getInstancesCount();
+
+    return (
+      <div className="tooltip-line-item">
+        <span>
+          {`${tasksRunning} ${StringUtil.pluralize("instance", tasksRunning)} running out of ${instancesTotal}`}
+        </span>
+      </div>
+    );
+  }
+
+  render() {
+    const { service } = this.props;
+    const instancesCount = service.getInstancesCount();
+    const serviceStatus = service.getServiceStatus();
+    const tasksRunning = service.getTaskCount();
+
+    if (
+      serviceStatus === ServiceStatus.RUNNING ||
+      serviceStatus === ServiceStatus.STOPPED ||
+      serviceStatus === ServiceStatus.NA
+    ) {
+      return null;
+    }
+
+    if (serviceStatus === ServiceStatus.DELETING) {
+      return (
+        <ProgressBar
+          className="status-bar--large"
+          data={[
+            {
+              className: "staged",
+              value: instancesCount
+            }
+          ]}
+          total={instancesCount}
+        />
+      );
+    }
+
+    return (
+      <Tooltip interactive={true} content={this.getTooltipContent()}>
+        <ProgressBar
+          className="status-bar--large staged"
```

Comment:
Did you mean to apply the `staged` class here?

Reply:
correct, since it's "default" now for the bar.